src/sentry/tasks/unmerge.py

```diff
@@ -0,0 +1,468 @@
+from __future__ import absolute_import
+
+import logging
+from collections import defaultdict
+
+from django.db.models import F
+
+from sentry.app import tsdb
+from sentry.constants import DEFAULT_LOGGER_NAME, LOG_LEVELS_MAP
+from sentry.event_manager import ScoreClause, generate_culprit, get_hashes_for_event, md5_from_hash
+from sentry.models import Environment, Event, EventMapping, EventTag, EventUser, Group, GroupHash, GroupRelease, GroupTagKey, GroupTagValue, Project, Release, UserReport
+
+
+def merge_mappings(values):
+    result = {}
+    for value in values:
+        result.update(value)
+    return result
+
+
+initial_fields = {
```

Comment:
These are all the fields that have last-write-wins semantics in `EventManager`, so they're written first and never updated. If other events are ingested while this backfill is occurring, the new events will overwrite these values which is intended.

I didn't make any editorial decisions about what fields to include here rather than as part of the backfill fields, so if you disagree with some of those choices, that's a discussion to have outside of the scope of this patch.